### DISCUSSION TITLE
Duplicate device identity when restoring a new device from old iCloud backup.

### DISCUSSION THREAD

**Discussion by tppq** [MobiusSync/MobiusSync#153] [Q&A] [answered]
I obtained a new iDevice, and to get started, restored an iCloud backup from an older device to it.

MöbiusSync seemed to restore everything from the old device, including the device ID. I think that this will eventually lead into something bad happening, considering that the old device is still around, and I suspect that Syncthing will get confused by multiple devices with same ID but different contents are around.

Thus, the **actual issue**: I think that device ID inside Möbius should change when underlying device changes. The  way forward could be to dig a genuine device ID every time Möbius starts, and if that has changed then take action of some kind (asking user what to do would be nicest, but of course is also needs more UI work).

How should I get around, while waiting for a possible proper fix? I would know what to do on command line, but am out of options in iWorld. Is there a way to force Möbius into changing the present DeviceID? I have the purchased version (or had on old device, unsure if purhases are tied to devices or iAccounts).

**Comment by SimonPickup** [comment, marked as answer]
You can delete the app and reinstall - which will force a new Device ID to be created. Purchase is tied to Apple ID.
